### ISSUE TITLE
Add HoloViews/Bokeh examples to pythonplot website

### ISSUE THREAD

**Issue by jbednar** [holoviz/holoviews#1623]
The [Python Plotting for EDA](http://pythonplot.com) website has examples of how to build various fairly simple plots using different Python plotting packages.  It's coming from an R background, so it seems to focus primarily on the sorts of plots that are well-supported by ggplots and related tools, and less on things that are better supported in Python.  Still, I think it would be useful to submit HoloViews-based plots to that website, showing that Bokeh plots can be constructed using HoloViews about as easily as for the ggplot-based versions.  We could then also submit some examples of things that would be much more difficult or unnatural to do in ggplot, to make the comparison a bit fairer.

**Comment by philippjfr** [comment]
I've made a start on this [here](https://anaconda.org/philippjfr/pythonplot-com/notebook). I've covered most things but I think we should address some of the issues I brought up there and wait until we've covered some of the remaining statistical operations.

**Comment by jbednar** [comment]
I don't think this was "completed", as I don't see anything at https://pythonplot.com about HoloViews or Bokeh. It's still appropriate to close it, as that site doesn't seem updated any more, but not as "completed".
